libpod/sqlite_state.go

```diff
@@ -32,10 +32,6 @@ type SQLiteState struct {
 const (
 	// Deal with timezone automatically.
 	sqliteOptionLocation = "_loc=auto"
-	// Set the journal mode (https://www.sqlite.org/pragma.html#pragma_journal_mode).
-	sqliteOptionJournal = "&_journal=WAL"
-	// Force WAL mode to fsync after each transaction (https://www.sqlite.org/pragma.html#pragma_synchronous).
-	sqliteOptionSynchronous = "&_sync=FULL"
```

Comment:
@vrothberg which flake are we talking about now? Because I'm not seeing the "stop" flake any more. Also, #17954 still has the sqlite settings untouched, and that PR sees the "stop" flake all the time.

Reply:
With only `&_sync=FULL` but without `WAL`, podman-info barfs instantly:
```
Error: checking if DB config table exists: 
    Invalid _synchronous: FULL_loc=auto,
       expecting value of '0 OFF 1 NORMAL 2 FULL 3 EXTRA'
```